qemu/tests/block_check_memory_leak.py

```diff
@@ -66,7 +66,7 @@ def _execute_io_in_guest():
     process.system_output(cp_cmd, shell=True)
     check_cmd = params["check_cmd"]
     out = process.system_output(check_cmd, shell=True).decode()
-    logger.info("Find leak:%s", out)
-
-    if len(out) and int(out) > 1000:
+    leak_threshold = params.get_numeric('leak_threshold')
+    logger.info("Find leak:%s,expect less than %d", out, leak_threshold)
```

Comment:
if leak_threshold = 0, the info message like "Find leak:0,expect less than 0", I think it's not suitable. I think it's OK that keep the previous statement unchanged. And it's better to output this info message when the test was failed. Just my suggestion, thanks.

Reply:
change to logger.info("Find leak:%s,threshold: %d", out, leak_threshold)